### PR TITLE
Feature/invitation

### PR DESCRIPTION
**Pull request by felyfely**
# Submit a pull request

## CLA

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [ ] The code changes follow best practices
- [ ] Code changes are tested (add some information if not applicable)

### CODE REVIEW

**Comment by tbarbugli** [comment]
@felyfely thank you for the contribution, we'll have a look at it and give some feedback as soon as possible :)

File: Sources/StreamChat/APIClient/Endpoints/ChannelEndpoints.swift [hunk 1]
```diff
@@ -67,6 +67,26 @@ extension Endpoint {
         )
     }
 
+    static func acceptInvite(cid: ChannelId, userId: String, message: String?) -> Endpoint<EmptyResponse> {
```
Comment (tbarbugli) [inline Sources/StreamChat/APIClient/Endpoints/ChannelEndpoints.swift:70]: the `userId` parameter is not needed, only server-side API clients use that

File: Sources/StreamChat/Models/Attachments/ChatMessageAudioAttachment.swift [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+/// A type alias for attachment with `AudioAttachmentPayload` payload type.
+///
+/// The `ChatMessageAudioAttachment` attachment will be added to the message automatically
+/// if the message was sent with attached `AnyAttachmentPayload` created with
+/// local URL and `.audio` attachment type.
+public typealias ChatMessageAudioAttachment = _ChatMessageAttachment<AudioAttachmentPayload>
+
+/// Represents a payload for attachments with `.media` type.
+public struct AudioAttachmentPayload: AttachmentPayload {
```
Comment (tbarbugli) [inline Sources/StreamChat/Models/Attachments/ChatMessageAudioAttachment.swift:15]: adding support for AudioAttachments looks very interesting and a nice addition to the SDK, would you mind opening a separate PR for this?

Comment (felyfely) [reply Sources/StreamChat/Models/Attachments/ChatMessageAudioAttachment.swift:15]: of course, I'll remove it when the draft is ready for review

File: Sources/StreamChat/APIClient/Endpoints/ChannelEndpoints.swift (hunk 1 above)
Comment (felyfely) [reply Sources/StreamChat/APIClient/Endpoints/ChannelEndpoints.swift:70]: Okay, I saw it in the android code, I was wondering why it is needed

**Comment by b-onc** [comment]
Closing this due to invitations being merged to the main branch and audio attachments coming soon.

**Pull request closed without merging**